tonic_validate/tests/test_scorer_litellm.py

```diff
@@ -0,0 +1,581 @@
+from typing import Dict, List
+
+import pytest
+from tonic_validate.classes import Benchmark, LLMResponse, Run
+from tonic_validate.classes.llm_response import CallbackLLMResponse
+from tonic_validate.metrics import (
+    AnswerConsistencyBinaryMetric,
+    AnswerConsistencyMetric,
+    AnswerContainsPiiMetric,
+    AnswerMatchMetric,
+    AnswerSimilarityMetric,
+    AugmentationAccuracyMetric,
+    AugmentationPrecisionMetric,
+    BinaryMetric,
+    ContainsTextMetric,
+    ContextContainsPiiMetric,
+    ContextLengthMetric,
+    DuplicationMetric,
+    HateSpeechContentMetric,
+    LatencyMetric,
+    RegexMetric,
+    ResponseLengthMetric,
+    RetrievalPrecisionMetric,
+)
+from tonic_validate import ValidateScorer
+
+
+def check_run(
+    expected_metric_scores: Dict[str, float],
+    scorer: ValidateScorer,
+    llm_responses: List[LLMResponse],
+) -> None:
+    run = scorer.score_responses(llm_responses)
+    for metric_name, expected_score in expected_metric_scores.items():
+        assert run.overall_scores[metric_name] == expected_score
+        for item in run.run_data:
+            assert item.scores[metric_name] == expected_score
+
+
+@pytest.mark.parametrize("answer, expected_score", [("Fido", 1.0), ("Rex", 0.0)])
+def test_answer_consistency_binary_metric(answer, expected_score):
```

Comment:
It seems this is a copy of the other test file. To save code, you can actually just use the other file and parametrize the fixture to provide different versions of the model evaluator. By doing that it will auto rerun all the tests for each llm model

https://docs.pytest.org/en/7.1.x/how-to/fixtures.html#fixture-parametrize